src/user/contributing.rst

```diff
@@ -49,4 +49,18 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
 
+
+.. _Guidelines:
+
+Writing Guidelines
+===========================
+
+Follow these guidelines for contributing or amending conda-forge documentation:
+
+1. For submitting or updating conda-build documentation, follow these rules.
+2. Determine who your target audience is, their level of expertise, and how they will use the content.
+3. Match the technical jargon to the audience's degree of understanding. It's preferable to underestimate rather than overestimate your readers' knowledge. Limit the usage of technical phrases to those that the user is likely to encounter. Use a glossary to augment definitions in the text if you need to define a large number of terms.
```

Comment:
Ooh, "augment" is a great word, but let's make this as accessible as possible. Maybe "supplement" or "enhance"?